build-system/base.js

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/** @const {!Object}  */
+const process = {};
+
+process.env = {};
+
+process.env.NODE_ENV = 'production';
```

Comment:
Is there any reason the above three lines can't be combined into one line like this?
```
const process = {
  env: {
    'NODE_ENV': 'production',
  },
};
```

Reply:
this is an extern file so we can't really write it like normal javascript

Reply:
oh wait this is just base.js my bad. yeah we can write it that way